src/components/Radios.tsx

```diff
@@ -0,0 +1,282 @@
+      hasConditional = true;
+    }
+  }
+
+  if (radiosHint) {
+    const hintId = radiosHint.id ? radiosHint.id : `${idPrefix}-hint`;
+    describedBy.push(hintId);
+    hintComponent = <Hint id={hintId} {...radiosHint} />;
+  }
+
+  if (errorMessage) {
+    const errorId = errorMessage.id ? errorMessage.id : `${idPrefix}-error`;
+    describedBy.push(errorId);
+    errorMessageComponent = <ErrorMessage id={errorId} {...errorMessage} />;
+    if (formGroup) {
+      formGroup.error = true;
+    } else {
+      formGroup = { error: true };
+    }
+  }
+
+  const extraAttributes = Attributes.ariaAndData(props);
+  const radioClassNames = classNames("govuk-radios lbh-radios", className, {
+    "govuk-radios--conditional": hasConditional
+  });
+
+  const renderDivider = (
+    item: RadioButton,
+    index: number,
+    radioExtraAttributes: React.AriaAttributes & DataAttributes
+  ): JSX.Element => {
+    return (
+      <div
+        className="govuk-radios__divider"
+        key={index}
+        {...radioExtraAttributes}
+      >
+        {item.divider}
+      </div>
+    );
+  };
+
+  const renderRadio = (
+    item: RadioButton,
+    index: number,
+    radioExtraAttributes: React.AriaAttributes & DataAttributes
+  ): JSX.Element => {
+    let id;
+    if (item.id) {
+      id = item.id;
+    } else {
+      if (item.value) {
+        id = `${idPrefix}-${item.value.replace(/\s+/g, "-").toLowerCase()}`;
+      } else {
+        id = `${idPrefix}-${index}`;
+      }
+    }
+    const hasHint = item.hint && item.hint.children ? true : false;
+    const itemHintId = `${id}-item-hint`;
+    const labelClasses = classNames("govuk-radios__label", {
+      [item.label?.className as string]: item.label && item.label.className
+    });
+    const conditionalClasses = classNames("govuk-radios__conditional", {
+      "govuk-radios__conditional--hidden": !item.checked
+    });
+
+    return (
+      <React.Fragment key={index}>
+        <div className="govuk-radios__item">
+          <input
+            className="govuk-radios__input"
+            id={id}
+            name={name}
+            type="radio"
+            value={item.value}
+            checked={item.checked ? true : undefined}
+            disabled={item.disabled ? true : undefined}
+            data-aria-controls={
+              item.childrenWhenChecked ? `conditional-${id}` : undefined
+            }
+            aria-describedby={hasHint ? itemHintId : undefined}
+            {...radioExtraAttributes}
+            onChange={
+              onChange !== undefined
+                ? (): void => {
+                    //eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                    onChange(item.value!);
+                  }
+                : undefined
+            }
+          />
+          <Label className={labelClasses} labelFor={id} {...item.label}>
+            {item.label && item.label.children}
+          </Label>
+          {hasHint && (
```

Comment:
```suggestion
          {item.hint?.children && (
```

Would be more idiomatic and reduce the amount of hunting to do in the code. React knows to ignore `undefined` elements.